extexpr/unwrap.go

```diff
@@ -43,6 +43,13 @@ func UnwrapFloat(expr parser.Expr) (float64, error) {
 	}
 }
 
+// UnsafeUnwrapFloat is like UnwrapFloat but should only be used in cases where the parser
+// guarantees success by already only allowing floats wrapped in parentheses.
+func UnsafeUnwrapFloat(expr parser.Expr) float64 {
```

Comment:
Should we inject this from `execution.go?`

Reply:
:heavy_check_mark: